src/Optimizer/LocalBundleAdjustment.cc

```diff
@@ -105,14 +100,6 @@ void Optimizer::LocalBundleAdjustment(KeyFrame* pKF, bool* pbStopFlag,
       }
     }
   }
-  num_fixedKF = lFixedCameras.size() + num_fixedKF;
-
-  if (num_fixedKF == 0) {
-    Verbose::PrintMess(
-        "LM-LBA: There are 0 fixed KF in the optimizations, LBA aborted",
-        Verbose::VERBOSITY_NORMAL);
-    return;
```

Comment:
Are you sure this if statement is always false?

Reply:
There's a good chance that if this condition were true, other segments of code would break before it got here, but I've reimplemented it just in case.